tests/acceptance/features/bootstrap/WebDavLockingContext.php

```diff
@@ -351,8 +351,13 @@ public function publicHasLockedFileLastSharedFolder(
 		string    $file,
 		TableNode $properties
 	) {
+		if ($this->featureContext->isUsingSharingNG()) {
+			$token = $this->featureContext->shareNgGetLastCreatedLinkShareToken();
+		} else {
+			$token = $this->featureContext->getLastCreatedPublicShareToken();
+		}
```

Comment:
```suggestion
		$token = ($this->featureContext->isUsingSharingNG()) ? $this->featureContext->shareNgGetLastCreatedLinkShareToken() : $this->featureContext->getLastCreatedPublicShareToken();
```

Reply:
I guess previous looked cleaner than this one.